packages/synapse-react-client/src/mocks/query/syn16787123.ts

```diff
@@ -1,869 +1,724 @@
 import { QueryResultBundle } from '@sage-bionetworks/synapse-types'
 
-const syn16787123: QueryResultBundle = {
+const syn16787123 = {
```

Comment:
I updated this mock to match the latest version of this table. The motivation to update was that I needed a result more than 25 rows for one of our existing tests, but we originally only saved 50 rows in this file.

I had to update some tests to use other mock query results, because this table no longer has faceted columns, so some tests were no longer valid against this table.